wizard/WizardCreateDevice1.qml

```diff
@@ -201,10 +201,94 @@ Rectangle {
+                hardwareWalletAsk.titleText = qsTr("Connect your Ledger, unlock it and open Monero app...") + translationManager.emptyString;
+            } else {
+                hardwareWalletAsk.titleText = qsTr("Connect and unlock your Trezor to continue...") + translationManager.emptyString;
+            }
+            hardwareWalletAsk.imgSource = hardwareWalletType == "Ledger" ? "qrc:///images/ledgerPIN.png" : "qrc:///images/trezorPIN.png";
+            hardwareWalletAsk.messageInstructions1 = "";
+            hardwareWalletAsk.messageInstructions2 = "";
+            hardwareWalletAsk.okButtonText = qsTr("Create wallet") + translationManager.emptyString;
+        } else {
+            hardwareWalletAsk.titleText = qsTr("Can't connect to %1").arg(hardwareWalletType) + translationManager.emptyString;
+            hardwareWalletAsk.imgSource = "";
+            if (hardwareWalletType == "Ledger") {
+                hardwareWalletAsk.messageInstructions1 = qsTr("Check if your Ledger has the Monero app installed (available in Ledger Live) and is running it.") + translationManager.emptyString;
+                hardwareWalletAsk.messageInstructions2 = qsTr("Check if your Ledger has the most recent firmware version (available on Manager section of Ledger Live).") + translationManager.emptyString;
+            } else {
+                hardwareWalletAsk.messageInstructions1 = qsTr("Check if the USB cable is connected. Push the cable into your Trezor until you hear/feel a click.") + translationManager.emptyString;
+                hardwareWalletAsk.messageInstructions2 = qsTr("Check if your Trezor has firmware v%1 or higher (available on Trezor Wallet or Trezor Suite).").arg(trezorFirmwareVersion) + translationManager.emptyString;
+            }
+            hardwareWalletAsk.okButtonText = qsTr("Try again") + translationManager.emptyString;
+        }
+
+        hardwareWalletAsk.onAcceptedCallback = function() {
+            wizardController.walletOptionsName = walletInput.walletName.text;
+            wizardController.walletOptionsLocation = walletInput.walletLocation.text;
+            wizardController.walletOptionsDeviceName = wizardCreateDevice1.deviceName;
+            if(lookahead.text)
+                wizardController.walletOptionsSubaddressLookahead = lookahead.text;
+            var _restoreHeight = 0;
+            if(restoreHeight.text){
+                // Parse date string or restore height as integer
+                if(restoreHeight.text.indexOf('-') === 4 && restoreHeight.text.length === 10){
+                    _restoreHeight = Wizard.getApproximateBlockchainHeight(restoreHeight.text, Utils.netTypeToString());
+                } else {
+                    _restoreHeight = parseInt(restoreHeight.text)
+                }
+                wizardController.walletOptionsRestoreHeight = _restoreHeight;
+            }
+            wizardController.walletCreatedFromDevice.connect(onCreateWalletFromDeviceCompleted);
```

Comment:
```suggestion
```

This code duplication seems unnecessary.

Reply:
This is a copy of onNextClicked of WizardNav in Line 171. I first tried to call it, but it returns a "not a function" error.